intercity-server.rb

```diff
@@ -20,11 +24,41 @@ def usage
     puts "Commands:"
     puts "    help - Show the commands available"
     puts "    install - Run the setup for installing intercity-server"
+    puts "    restart - Restart your Intercity instance"
+    puts "    update - Update your Intercity instance"
   end
 
   def install
+    if installed?
+      puts "Intercity is already installed."
```

Comment:
@michiels Thats the reason I want people to be aware of the CLI. That is stuff I want to be done over the CLI. Not via the bash script. This change has nothing to do with the initial bash script. This is the CLI